xysched/README.md

```diff
@@ -1,18 +1,23 @@
 # Introduction
+
 Xysched provides a mechanism of job scheduling in future with a simple syntax.
 
 # Features
+
 There are two most important objects in this library:
-- `Future` defines tasks.
-- `Scheduler` manages to schedule `Future` instances.
+
+-   `Future` defines tasks.
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/xybor/xyplatform/pullRequest?prid=10050498)
```suggestion
- `Future` defines tasks.
```